README.md

```diff
@@ -267,7 +272,7 @@ To build the application in your system, you'll need to install the dependencies
 
 ```shell
 # Compile-time
-apt install g++ build-essential qt5-default qt5-qmake qttools5-dev-tools
```

Comment:
Please also drop qt5-qmake since it is not needed anymore.

Reply:
removed